### PR TITLE
To Do list: List Structure.

### PR DESCRIPTION
**Pull request by Hisoka37**
#  In this PR, I implemented the following:

- [ ] Set up a new project with webpack that is based on the webpack exercise you have already completed.
- [ ] Create an index.html file and write my HTML markup. Create an empty To Do List placeholder (`<div>` or `<ul> `element). 
     The index.html file must be set as a template using the HTML Webpack Plugin.
- [ ] Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- [ ] Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,134 @@
...
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+- **[future_feature_1]**  
+- **[future_feature_2]**  
+- **[future_feature_3]** 
+- **[future_feature_4]** 
+- **[future_feature_5]** 
+
```
Comment (muneebulrehman) [inline README.md]: Kindly add the heading ** Future Features ** to this section and also list 1-3 potential features to be added in the future.

File: README.md [hunk 2]
```diff
@@ -1 +1,134 @@
...
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (muneebulrehman) [inline README.md]: The link to the MIT license is broken. Kindly fix this issue.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Hisoka37 , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- All technical requirements met for this milestone.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,131 @@
...
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+I would like to thank  Microverse for the opportunity and motivation.
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
```
Comment (Shedrack-Sunday) [inline README.md]: - [Optional] Kindly remove this section since it's not implemented, or complete it to make your work more professional 👍

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @Hisoka37 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


# Highlights
- No linter errors ✔️
- Good implementations ✔️

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**